### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kwacky1**
Tracker ID: #ADD LINK TO PIVOTAL STORY

Unit tests completed?: (Y/N)

PR Branch #ADD LINK TO PR BRANCH

Code Coverage & Build Info #ADD LINK TO JENKINS CONSOLE

E2E Approved #ADD LINK TO PASSING E2E TESTS

Windows Testing #HAS WINDOWS BEEN TESTED?

Related PR #ADD ANY RELATED PULL REQUESTS

UX Approved #ADD UX APPROVAL IF NEEDED

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,13 @@
...
+a pull request
+with
+multiple lines
+of code
+
 Pull Request Templates are fun.
 
 Approvals are even more fun.
 
 Am I a first time contributor?
```
Comment (kwacky1) [inline README.md]: comment

**Comment by kwacky2** [comment]
we need more comments

**Comment by kwacky2** [comment]
> we need more comments

how many more comments?

**Comment by kwacky2** [comment]
> > we need more comments
> 
> how many more comments?

That should be enough

**Comment by kwacky2** [comment]
Need some fresh notifications

**Comment by kwacky2** [comment]
> Need some fresh notifications

Three seems to be the magic number

**Comment by kwacky2** [comment]
> > Need some fresh notifications
> 
> Three seems to be the magic number

Just to be safe

**Pull request closed without merging**